src/popup/bookmarkModule.js

```diff
@@ -304,6 +304,10 @@ document.addEventListener('DOMContentLoaded', () => {
     e.target.style.display = 'none';
 
     removeBookmarkImages();
+
+    if (localStorage !== null) {
```

Comment:
What does this condition do? Even after removing it, I have not seen any change in the behavior of the extension.